llvm/test/Transforms/LoadStoreVectorizer/X86/massive_indirection.ll

```diff
@@ -0,0 +1,63 @@
+; RUN: opt %s -mtriple=x86_64-unknown-linux-gnu -passes=load-store-vectorizer -mcpu=skx -S -o %t.out.ll
+; RUN: FileCheck -input-file=%t.out.ll %s
```

Comment:
Avoid the temporary file, nearly all tests should just pipe directly to FileCheck

Reply:
Fixed as suggested: https://github.com/llvm/llvm-project/pull/114501/commits/427983a9378f8612c054ad219023564ecf803643